package.js

```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: 'ethereum:tools',
   summary: 'Helper functions for dapps',
-  version: '0.7.0',
+  version: '1.0.0',
```

Comment:
Right to one-dot-oh eh? Balsy. 😜

Reply:
Haha, well I thought it requires a major version bump. In meteor you can’t have packages with different versions of the same packages so it seems breaking, but happy to do whatever is most appropriate.